.vscode/settings.json

```diff
@@ -1,3 +1,12 @@
 {
-  "editor.formatOnSave": true
+  "editor.defaultFormatter": "esbenp.prettier-vscode",
```

Comment:
Not sure if we need to check in our vscode editor settings? @iharmanpannu Am I missing something here ?

Reply:
Fixed